.gitignore

```diff
@@ -2,8 +2,9 @@ node_modules
 .DS_Store
 
 # Build products
-component.js
-function.js
-index.js
-mixin.js
-shallowEqual.js
\ No newline at end of file
+/component.js
+/decorator.js
+/function.js
+/index.js
+/mixin.js
+/shallowEqual.js
```

Comment:
This pattern is explicitly documented in the Git book: http://git-scm.com/docs/gitignore

>A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".

And `./` doesn't work - I just checked.